From: Sergio Costas <rastersoft@gmail.com>
Date: Mon, 15 Aug 2022 15:35:06 +0200
Subject: Remove unneeded unblock

When a signal handler is disconnected, it doesn't matter if it
was blocked or not, so there's no need to unlock it before
disconnection.

Origin: upstream, 3.24.35, commit:02ea88bba2169e4a6d3bcc2dda25133a0ec5f45b
---
 gtk/gtkpopover.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gtk/gtkpopover.c b/gtk/gtkpopover.c
index 312d933..4182c5f 100644
--- a/gtk/gtkpopover.c
+++ b/gtk/gtkpopover.c
@@ -2127,7 +2127,6 @@ gtk_popover_update_relative_to (GtkPopover *popover,
         {
           if (priv->grab_notify_blocked)
             {
-              g_signal_handler_unblock (priv->widget, priv->grab_notify_id);
               priv->grab_notify_blocked = FALSE;
               had_grab_notify_blocked = TRUE;
             }
